src/actions/actions.ts

```diff
@@ -19,7 +19,57 @@
 // THE SOFTWARE.
 
 import ActionTypes from 'constants/action-types';
-import {createAction} from 'redux-actions';
+import {createAction} from '@reduxjs/toolkit';
+
+import {ParsedConfig} from '../schemas';
+import {RGBColor} from 'reducers/types';
+import {Bounds} from 'reducers/map-state-updaters';
+import {MapInfo} from 'reducers/vis-state-updaters';
+import {UiState} from 'reducers/ui-state-updaters';
+
+/**
+ * Input dataest parsed to addDataToMap
+ */
+export type ProtoDataset = {
+  info: {
+    id?: string;
+    label?: string;
+    format?: string;
+    color?: RGBColor;
+  };
+  data: {
+    fields: {
+      name: string;
+      type?: string;
+      format?: string;
+      displayName?: string;
+      id?: string;
+    }[];
+    rows: any[][];
+  };
+
+  // table-injected metadata
+  metadata?: any;
+  supportedFilterTypes?: string[];
+};
+
+export type AddDataToMapOptions = {
```

Comment:
Add a JSDoc header for each type.
```suggestion
/** Options for addDataToMap() */
export type AddDataToMapOptions = {
```